packages/react-ui/src/Titles/Heading/index.js

```diff
@@ -4,9 +4,16 @@ import styled, { css } from "styled-components";
 import { Stripe } from "../../Stripe";
 import { breakpoints, fonts, spacings } from "../../theme";
 
-export const Heading = ({ children, stripped = false, ...props }) => (
+export const Heading = ({ children, stripped = false, variant, ...props }) => (
   <StyledHeading stripped={stripped} {...props}>
-    {stripped && <Stripe variant="primary" position="left" length="100%" />}
+    {stripped && (
+      <Stripe
+        rounded={variant !== "primary"}
+        variant={variant}
+        position="left"
+        length="100%"
```

Comment:
is this css ?

Reply:
Sort of, but it can be either width or height depending on the stripe's position

Reply:
Didn't catch that :/ 
Is it validate using  propTypes  ? I guess position is an enum 
Does it work if I put length="3rem"

Reply:
Yep

Reply:
https://socialgouv-react-ui.netlify.com/?path=/story/components-stripe--base